File: src/main/java/com/bajdcc/LALR1/interpret/os/user/routine/URCount.java
package com.bajdcc.LALR1.interpret.os.user.routine;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【用户态】计数
 *
 * @author bajdcc
 */
public class URCount implements IOSCodePage {
	@Override
	public String getName() {
		return "/usr/p/count";
	}

	@Override
	public String getCode() {
		return ResourceLoader.load(getClass());
	}
}

File: src/main/java/com/bajdcc/LALR1/interpret/os/user/routine/URCount.kt
package com.bajdcc.LALR1.interpret.os.user.routine

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【用户态】计数
 *
 * @author bajdcc
 */
class URCount : IOSCodePage {
    override val name: String
        get() = "/usr/p/count"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
